hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -270,6 +270,36 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     }
   }
 
+  private BucketLayout getDefaultBucketLayout(OzoneManager ozoneManager,
+      BucketInfo bucketInfo, String volumeName, String bucketName) {
+
+    if (getOmRequest().getVersion() <
+        ClientVersion.BUCKET_LAYOUT_SUPPORT.toProtoValue()) {
+
+      if (LOG.isDebugEnabled()) {
```

Comment:
This should mostly be a LOG.info message. or we should add a metric around it to identify cases where buckets are created from older client.

Reply:
Thanks @mukul1987 . Done!